activiti-cloud-starter-runtime-bundle/src/test/resources/processes/task-date-var-mapping-extensions.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id":"taskDateVarMapping",
+  "name":"myProcess.bpmn",
+  "extensions": {
+    "properties": {
+     "process-variable-string-id": {
+        "id": "process-variable-string-id",
+        "name": "process-variable-string",
+        "type": "string",
+        "required": true,
+        "value": "stringValue"
+      },
+      "process-variable-integer-id": {
+        "id": "process-variable-integer-id",
+        "name": "process-variable-integer",
+        "type": "integer",
+        "required": true,
+        "value": 1
+      },
+      "process-variable-boolean-id": {
+        "id": "process-variable-boolean-id",
+        "name": "process-variable-boolean",
+        "type": "boolean",
+        "required": true,
+        "value": true
+      },
+      "process-variable-date-id": {
+        "id": "process-variable-date-id",
+        "name": "process-variable-date",
+        "type": "date",
+        "required": true,
+        "value": "2019-09-01"
+      },
+      "process-variable-datetime-id": {
+        "id": "process-variable-datetime-id",
+        "name": "process-variable-datetime",
+        "type": "date",
+        "required": true,
+        "value": "2019-09-01"

```

Comment:
Now you can specify a date with time here, right?

Reply:
Yes we can... After corresponding PR (activiti-core) was merged, it is possible... Should I apply this here? This check was made in other PR...

Reply:
Please, apply it here as well.. No need to to check the actual value, just change the format (for consistency). You can use zeros to keep the same date if you prefer.

Reply:
Changed